keras_eval/eval.py

```diff
@@ -176,28 +175,31 @@ def save_probabilities_labels(self, id, save_path):
 
     def compute_inference_probabilities(self, concept_dictionary, probabilities):
         '''
-        Combines probabilities based on key "group" in concept_dictionary and saves the values in self.probabilities
+        Computes the probability inference based on key "group" in concept_dictionary and saves the values in self.probabilities
 
         Args:
             concept_dictionary: It is the dictionary which contains all the granular concepts and the mapping with the groups.
             probabilities: These are computed granular probabilities
 
         '''
-
         for concept in concept_dictionary:
-
             if concept['group'] in self.group_id_dict.keys():
                 self.group_id_dict[concept['group']].append(concept['class_index'])
             else:
                 self.group_id_dict[concept['group']] = [concept['class_index']]
 
-        self.combined_probs = [[[0.0, ] * len(self.concept_labels)] * len(probabilities[0])]
-        self.combined_probs = np.array(self.combined_probs)
-        for idx, concept_label in enumerate(self.concept_labels):
-            column_numbers = self.group_id_dict[concept_label]
-            for column_number in column_numbers:
-                self.combined_probs[0][:, idx] += probabilities[0][:, column_number]
-        self.probabilities = self.combined_probs
+        inference_probs_list = []
+        inference_probs = np.zeros((1, len(combined_probabilities), len(concept_labels)))
```

Comment:
Where are you assigning `combined_probabilities` and `concept_labels`

Reply:
@jsalbert sorry, the PR is one commit behind. I'll request your review when fixed shortly 🙏